Production_scripts/Process_extracts/extract_cmh.R

```diff
@@ -98,6 +98,7 @@ cmh_clean <- cmh_extract %>%
 
 outfile <- cmh_clean %>%
   select(
+    year,
```

Comment:
The script didn't select `year` before and I assumed this was an oversight as `year` was in the existing data